docrawl/docrawl_client.py

```diff
@@ -90,15 +94,15 @@ def _wait_until_page_is_loaded(self, timeout=20):
             except:
                 is_page_loaded = False
             time.sleep(0.5)
-            docrawl_logger.info('Page is still loading, waiting 0.5 sec ...')
+            # docrawl_logger.info('Page is still loading, waiting 0.5 sec ...')
```

Comment:
dont disable this :) (change your log level)

Reply:
How can I change a log level for `docrawl_logger`?

Reply:
You need to implement it :)

Reply:
But I might unify it with Flog